lib/pronto/formatter/pull_request_formatter.rb

```diff
@@ -0,0 +1,87 @@
+module Pronto
+  module Formatter
+    class PullRequestFormatter
```

Comment:
To clear up any confusion maybe this should be something like: `PullRequestFormatterBase` or `AbstractPullRequestFormatter`.